components/style/properties.mako.rs

```diff
@@ -178,7 +184,7 @@ pub mod longhands {
 
     <%def name="raw_longhand(name, **kwargs)">
     <%
-        products = kwargs.pop("products", "gecko,servo")
+        products = kwargs.pop("products", "gecko servo")
```

Comment:
Nit: Even though it doesn't really matter, I think we should add a .split() on the rhs to be more consistent with the other patterns in this file. Thanks for fixing this!